zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/archiver/OpenSearchRepository.java

```diff
@@ -195,6 +193,45 @@ public CompletableFuture<Void> reindexDocuments(
         .thenApplyAsync(ignored -> null, executor);
   }
 
+  private List<String> fetchIndexMatchingIndexes(final String indexWildCard) throws IOException {
+    final List<String> indices = new ArrayList<>();
+    client
+        .cat()
+        .indices()
+        .thenAccept(
+            response ->
+                response.valueBody().stream()
+                    .map(IndicesRecord::index)
+                    .filter(index -> index.matches(indexWildCard))
```

Comment:
:wrench: `String#matches` will convert this to a compiled `Pattern` instance. Since our index never changes, we can already compile it ourselves once, and then just use to match.

```suggestion
  final var matcher = Pattern.compile(indexWildCard).asMatchPredicate();

  client
        .cat()
        .indices()
        .thenAccept(
            response ->
                response.valueBody().stream()
                    .map(IndicesRecord::index)
                    .filter(matcher)
```